### PR TITLE
Cover guis

### PR DESCRIPTION
**Pull request by moller21**
Added all used covers i think, since i started it i guess i had to finish it.

Worth noting that I changed the way the arm & the item detector cover works, so those 2 would have to be reconfigured, rest works the same.

Not entirely sure how safe the moveFromSlotToSide() function is, worth testing some more but it's not like anyone uses the arms.

### CODE REVIEW

File: src/main/java/gregtech/api/util/GT_Utility.java [hunk 1]
```diff
@@ -665,6 +665,64 @@ public static byte moveFromSlotToSlot(IInventory aTileEntity1, IInventory aTileE
         return 0;
     }
 
+    /**
+     * Moves Stack from Inv-Side to Inv-Slot.
+     *
+     * @return the Amount of moved Items
+     */
+    public static byte moveFromSlotToSide(IInventory fromTile, Object toTile, int aGrabFrom, byte aPutTo, List<ItemStack> aFilter, boolean aInvertFilter, byte aMaxTargetStackSize, byte aMinTargetStackSize, byte aMaxMoveAtOnce, byte aMinMoveAtOnce, boolean aDoCheckChests) {
```
Comment (draknyte1) [inline src/main/java/gregtech/api/util/GT_Utility.java:673]: moveFromSlotToSide Won’t work for things with > 128 slots. 
Also won’t work for things moving more than 2 stacks at a time.

Comment (moller21) [reply src/main/java/gregtech/api/util/GT_Utility.java:673]: Could you explain why? It puts from specific slot, to a side of a tile. The side is a byte, slot is an int.

Also this is just mostly copied from the other functions, so should work the same as them, don't think they can move more than a stack at a time either, but not intended to.

**Comment by moller21** [comment]
> Breaking existing covers is a flat no. Not acceptable.

I changed the arm so you're able to specify the slot to grab from and/or put to. Before you could only specify what slot to put into. To make them more flexible. But there's a free bit, could toggle that to it work, unless it's set to grab from like slot# 2M.

The item meter works the same unless its specified a slot.

File: src/main/java/gregtech/common/covers/GT_Cover_EUMeter.java [hunk 2]
```diff
@@ -141,4 +147,109 @@ public boolean manipulatesSidedRedstoneOutput(byte aSide, int aCoverID, int aCov
...
+
+        public GUI(byte aSide, int aCoverID, int aCoverVariable, ICoverable aTileEntity) {
+            super(aTileEntity, 176, 107, GT_Utility.intToStack(aCoverID));
+            this.side = aSide;
+            this.coverID = aCoverID;
+            this.coverVariable = aCoverVariable;
+
+            GuiButton b;
+            b = new GT_GuiIconCheckButton(this, 0, startX + spaceX*0, startY+spaceY*1, GT_GuiIcon.CHECKMARK, null)
+                    .setTooltipText(trans("031", "Normal Universal Storage"));
```
Comment (Glease) [inline src/main/java/gregtech/common/covers/GT_Cover_EUMeter.java:183]: It might be better to remove these "Normal".

Comment (moller21) [reply src/main/java/gregtech/common/covers/GT_Cover_EUMeter.java]: good catch^^

**Pull request merged**